### ISSUE TITLE
[4.0] Extension update check

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#17844]
### Steps to reproduce the issue
With current J4 the extension update check never completes. Looking at the console is the following error
```
 Uncaught SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at onSuccess (extensionupdatecheck.js?61f0bd7f3a1e51d20f4535590f3e4657:28)
    at XMLHttpRequest.o.onreadystatechange (core.min.js?61f0bd7f3a1e51d20f4535590f3e4657:1) 

```

@dgt41

**Comment by C-Lodder** [comment]
@brianteeman - could you replace the minified code with the uncompressed code and try again, to get the exact line number?

**Comment by dgrammatiko** [comment]
@C-Lodder its not the javascript, the error comes from php, the returned data is not JSON formatted. Something throws an error and the return data is html formatted

**Comment by dgrammatiko** [comment]
@brianteeman check #17846

**Comment by ghost** [comment]
closed as having Pull Request #17846<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17844">issues.joomla.org/tracker/joomla-cms/17844</a>.</sub>

**Comment by dgrammatiko** [comment]
@franz-wohlkoenig can you re open this one?
The PR was wrong
The error is coming from:
![screen shot 2017-09-03 at 15 05 12](https://user-images.githubusercontent.com/3889375/30002730-77920a18-90b9-11e7-8df1-6c72a02060a4.png)

**Comment by ghost** [comment]
reopened as stated above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17844">issues.joomla.org/tracker/joomla-cms/17844</a>.</sub>

**Comment by mbabker** [comment]
It is already fixed in staging so either the fix has not yet been merged to 4.0 or it was removed in a merge conflict.

**Comment by dgrammatiko** [comment]
Actually re installing a fresh copy resolves the issue
